src/software/thunderscope/common/common_widgets.py

```diff
@@ -115,7 +104,9 @@ def setValue(self, value):
         """
         super(ColorProgressBar, self).setValue(value * self.decimals)
 
-        percent = (self.value() - self.minimum()) / (self.maximum() - self.minimum())
+        percent = max(
+            0, (self.value() - self.minimum()) / (self.maximum() - self.minimum())
+        )
```

Comment:
if value() is greater than maximum() (not sure if it's possible, but could be if the maximum is not accurate...) then this will be greater than one. We should probably clamp it to max 1 as well to be safe